include/vkgcDefs.h

```diff
@@ -254,6 +254,13 @@ enum class ShadowDescriptorTableUsage : unsigned {
   Disable = 2,
 };
 
+/// Represent the features of VK_EXT_robustness2
+struct ExtendedRobustness {
+  bool robustBufferAccess; ///< robustBufferAccess2
```

Comment:
robustBufferAccess2 -> Robust buffer access, ...
Add more contents to explain what the feature is.

Reply:
Done.